TeamCode/src/main/java/com/acmerobotics/relicrecovery/drive/MecanumDrive.java

```diff
@@ -124,6 +126,11 @@ public static Pose2d getDelta (int[] rot) {
         return new Pose2d(x, y, h);
     }
 
+    public double ticksToRadians(int motor, int ticks) {
+        double ticksPerRev = motors[motor].getMotorType().getTicksPerRev();
```

Comment:
where do we set the gearbox so that getMotorType() returns the correct thing?

Reply:
Oh good point. Looking at the source, there is a `gearing` parameter but it isn't factored in to the `ticksPerRev` calculation. There are three main solutions:
* Use `MotorConfigurationType#setTicksPerRev()`
* Declare a new motor configuration type and declare it in the configuration (see `NeveRest20Gearmotor`)
* Use some other mechanism (e.g. declare a constant or add it as a field to a configuration object).